packages/frontend/src/common/hooks/useValidatedTimeInput.ts

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { isNumber } from "lodash";
+
+interface useValidatedTimeInputOptions {
+  formatLength?: number;
+  minValue?: number;
+  maxValue?: number;
+  initialValue?: number;
+}
+export function useValidatedTimeInput(options: useValidatedTimeInputOptions | undefined = {}) {
```

Comment:
I'd destructure the object here

```
export function useValidatedTimeInput({
  initialValue,
  formatLength,
  maxValue = Number.MAX_VALUE,
  minValue = Number.MIN_VALUE,
}: useValidatedTimeInputOptions | undefined = {}) {
```

and isValid could be implemented like this then:
```
  function isValid() {
    return value >= minValue && value <= maxValue;
  }
```

or are there other reasons this might not be valid?

Reply:
I see so the isValid method gets clearer and max, minValue get defaults right ahead. Code is changed